src/Notification/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import withStyles from '../../util/withStyles';
+import styles from './index.scss';
+
+const Notification = ({ type, msg, children, className, action }) => {
+  if (!(msg || children)) return null;
+  const content = children || msg;
+  const classes = classNames(
+    {
+      alert: true
+    },
+    `alert--${type}`,
```

Comment:
Couldn't you just replace this with `alert alert--${type}` and delete the 3 lines above?